orttraining/orttraining/eager/opgen/opgen/generator.py

```diff
@@ -1,6 +1,8 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
+# pylint: too-many-nested-blocks
```

Comment:
I believe this needs `disable=` ahead of the `too`

Reply:
please refresh as i updated it to `# pylint: disable=missing-docstring, too-many-public-methods, too-many-nested-blocks`